ocean_lib/data_provider/data_service_provider.py

```diff
@@ -812,6 +818,35 @@ def check_asset_file_info(did: str, provider_uri: str) -> bool:
 
         return False
 
+    @staticmethod
+    @enforce_types
+    def get_asset_urls(
+        did: str, service_id: str, provider_uri: str, wallet: Wallet
+    ) -> None:
+        if not did:
+            return False
```

Comment:
This error returns `False`. but the error on line 846 returns `None`.  Please pick one error convention.